packages/postgrest/lib/src/postgrest_builder.dart

```diff
@@ -317,18 +317,19 @@ class PostgrestBuilder<T, S, R> implements Future<T> {
   ///
   /// [url] may be used to update based on a different url than the current one
   Uri appendSearchParams(String key, String value, [Uri? url]) {
-    final searchParams = Map<String, dynamic>.from(_url.queryParametersAll);
+    final searchParams =
+        Map<String, dynamic>.from((url ?? _url).queryParametersAll);
     searchParams[key] = [...searchParams[key] ?? [], value];
     return (url ?? _url).replace(queryParameters: searchParams);
   }
 
   /// Get new Uri with overridden queryParams
   ///
   /// [url] may be used to update based on a different url than the current one
-  Uri overrideSearchParams(String key, String value, [Uri? url]) {
+  Uri overrideSearchParams(String key, String value) {
```

Comment:
Removed the `url` parameter since it was not used anywhere.

Reply:
I think we should keep it to allow these similar methods to have the same parameters. But I see that's not really important.

Reply:
I personally think we can just add it back when we actually need it. I think it creates extra confusion for having a piece of code that is not being used.